### ISSUE TITLE
obj_ctl: seg. fault. when loading external cfg. with PMEMOBJ_CONF_FILE that points to invalid file

### ISSUE THREAD

**Issue by kkajrewicz** [pmem/issues#521]
Steps to reproduce:
```
1. export PMEMOBJ_CONF_FILE="invalid_path"
2. Create obj pool with pmemobj_create("/tmp/mypool", NULL, PMEMOBJ_MIN_POOL, 644)
```
Logs:
```
(...)
<libpmemobj>: <1> [obj.c:188 obj_ctl_init_and_load] unable to parse config stored in invald file (from PMEMOBJ_CONF_FILE environment variable)
Segmentation fault (core dumped)
```
Notice also typo in dbg. log: 
`unable to parse config stored in **invald** file`

Found on 1.2+b1-179-g0fe628d

**Comment by pbalcer** [comment]
this is fixed in pmem/nvml#1826

**Comment by kkajrewicz** [comment]
Verified on 1.2+b1-253-g0dd3435
